fighters/common/src/general_statuses/cliff.rs

```diff
@@ -98,4 +99,26 @@ unsafe fn status_end_CliffJump2(fighter: &mut L2CFighterCommon) -> L2CValue {
 unsafe fn status_end_CliffJump3(fighter: &mut L2CFighterCommon) -> L2CValue {
     VarModule::set_vec3(fighter.object(), vars::common::instance::LEDGE_POS, Vector3f {x: 0.0, y: 0.0, z: 0.0});
     call_original!(fighter)
+}
+
+#[skyline::hook(replace = smash::lua2cpp::L2CFighterCommon_sub_cliff_uniq_process_exit_Common)]
+unsafe fn sub_cliff_uniq_process_exit_Common(fighter: &mut L2CFighterCommon, isleavecliff: L2CValue) {
+    let is_leave_cliff = isleavecliff.get_i32();
```

Comment:
`isleavecliff` as you call it isn't an integer, it's a boolean. The values passed into `set_xlu_frame_global` are supposed to be 0.

Reply:
> `isleavecliff` as you call it isn't an integer, it's a boolean. The values passed into `set_xlu_frame_global` are supposed to be 0.

You're right, my b, I misunderstood the decomp